bin/alluxio-start.sh

```diff
@@ -47,9 +44,6 @@ MOPT (Mount Option) is one of:
            specified directory (relative to each worker node's host filesystem).
 -f         format Journal, UnderFS Data and Workers Folder on master.
 -h         display this help.
--i backup  a journal backup to restore the master from. The backup should be
```

Comment:
if you remove the `-i` from usage description, it should also be removed from the opts parsing under `main() {` as well as any logic parts that it affects

Reply:
Good catch, updated